support-frontend/assets/pages/[countryGroupId]/components/checkoutComponent.tsx

```diff
@@ -347,13 +346,6 @@ export function CheckoutComponent({
 		}
 	}, [stripeExpressCheckoutSuccessful]);
 
-	/**
-	 * flag that disables the submission of the form until the stripeClientSecret is
-	 * fetched from the Stripe API with using the reCaptcha token
-	 */
-	const [stripeClientSecretInProgress, setStripeClientSecretInProgress] =
```

Comment:
No longer needed as this now happens when the form is submitted, not when the recaptcha is checked.